src/azure-cli/azure/cli/command_modules/appservice/_params.py

```diff
@@ -391,6 +395,8 @@ def load_arguments(self, _):
                    help='the container registry server username')
         c.argument('registry_password', options_list=['--registry-password', '-p', c.deprecate(target='--docker-registry-server-password', redirect='--registry-password')],
                    help='the container registry server password')
+        c.argument('min_replicas', options_list=['--min-replicas'], type=int, help="The minimum number of replicas when create funtion app on container app", is_preview=True)
+        c.argument('max_replicas', options_list=['--max-replicas'], type=int, help="The maximum number of replicas when create funtion app on container app", is_preview=True)
```

Comment:
Please do not need to explicitly specify the default option name repeatedly
```suggestion
        c.argument('min_replicas', type=int, help="The minimum number of replicas when create funtion app on container app", is_preview=True)
        c.argument('max_replicas', type=int, help="The maximum number of replicas when create funtion app on container app", is_preview=True)
```